src/xhr.js

```diff
@@ -57,14 +57,14 @@ const callbackWrapper = function(request, error, response, callback) {
 
 const xhrFactory = function() {
   const xhr = function XhrFunction(options, callback) {
-    // Add a default timeout for all hls requests
+    // Add a default timeout
     options = mergeOptions({
       timeout: 45e3
     }, options);
 
     // Allow an optional user-specified function to modify the option
     // object before we construct the xhr request
-    const beforeRequest = XhrFunction.beforeRequest || videojs.Hls.xhr.beforeRequest;
+    const beforeRequest = XhrFunction.beforeRequest || videojs.Vhs.xhr.beforeRequest;
```

Comment:
I'm not sure changing an option like this will be great for users. Perhaps we need to keep `videojs.Hls` around just for this property, and log a deprecation? We can still prioritize using `videojs.Vhs.xhr.beforeRequest`

Reply:
`videojs.Hls` is still present.